setup.py

```diff
@@ -16,6 +16,7 @@
 extra_reqs = {
     "dev": ["pytest", "pytest-cov", "pytest-asyncio", "pre-commit"],
     "stac": ["stac-tiler~=0.0rc2"],
+    "mosaic": ["cogeo-mosaic==3.0a4", "rio-tiler-mosaic"],
```

Comment:
`rio-tiler-mosaic` is not needed here

Reply:
Removed with https://github.com/developmentseed/titiler/pull/40/commits/097927aaaa59b9a942d4a84e3e9ea31ef982a808